SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -181,18 +180,11 @@ void USpatialGDKEditorSettings::SetNumberOfSimulatedPlayers(uint32 Number)
 
 bool USpatialGDKEditorSettings::IsDeploymentConfigurationValid() const
 {
-	bool result = IsAssemblyNameValid(AssemblyName) &&
-		IsDeploymentNameValid(PrimaryDeploymentName) &&
-		!GetSnapshotPath().IsEmpty() &&
-		!GetPrimaryLanchConfigPath().IsEmpty() &&
-		IsRegionCodeValid(PrimaryDeploymentRegionCode);
+	bool result = IsAssemblyNameValid(AssemblyName) && IsDeploymentNameValid(PrimaryDeploymentName) && !GetSnapshotPath().IsEmpty() && !GetPrimaryLanchConfigPath().IsEmpty() && IsRegionCodeValid(PrimaryDeploymentRegionCode);
```

Comment:
😬

Reply:
You have column limit set to 0 so I don't understand why this is changed. Perhaps, need to re-apply work in progress changes.